src/Xamarin.Android.Build.Tasks/Tasks/GetMonoPlatformJars.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (C) 2011 Xamarin, Inc. All rights reserved.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tasks
+{
+	public class GetMonoPlatformJars : Task
+	{
+		[Required]
+		public string TargetFrameworkDirectory { get; set; }
```

Comment:
Should this instead be a `string[]` or `ITaskItem[]` so that *MSBuild*` is responsible for the `;`-splitting?  Or would this not work like I think?

Reply:
It should work that way, I'll give it a try.